### DISCUSSION TITLE
Hi all I am new to nocodb and currently [Apr 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1943] [Discord Threads]
> 🙋‍♂️ **unkn**

**Comment by o1lab** [comment]
[Autogenerated from discord](https://discord.com/channels/661905455894888490/967240840554614835)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
